modules/ftui/ftui.binding.js

```diff
@@ -83,7 +102,7 @@ export class FtuiBinding {
       const filteredValue = this.filter(attributeValue, options.filter);
       const value = String(options.value).replace(/\$value/g, filteredValue);
       const [parameterId, deviceName, readingName] = ftuiHelper.parseReadingId(readingId);
-      console.log(options.cmd + '#' + deviceName+ '#' + readingName+ '#' + value)
+      console.log(options.cmd + '#' + deviceName + '#' + readingName + '#' + value)
```

Comment:
please remove all console.log before commit changes, even if I accidentally left it in.

Reply:
Removed - it was the only one in this module.